### PR TITLE
IWF-324: adding keyword array to search attributes

### PR DESCRIPTION
**Pull request by ktrops**
### Checklist
- [x] Code compiles correctly
- [x] Tests for the changes have been added
- [x] All tests passing
- [x] **This PR change is backwards-compatible**
- [ ] **This PR CONTAINS a (planned) breaking change** (it is not backwards compatible)

### Related Issue
Closes #issue_number

### CODE REVIEW

File: docker-compose/init-compose.sh [hunk 1]
```diff
@@ -19,7 +19,9 @@ for run in {1..60}; do
   sleep 0.1 
   temporal  operator search-attribute  create --name IwfExecutingStateIds --type KeywordList
   sleep 0.1
-  if checkExists "IwfWorkflowType" ] && checkExists "IwfGlobalWorkflowVersion" && checkExists "IwfExecutingStateIds" ] ; then
+  temporal  operator search-attribute  create --name CustomKeywordArrayField --type KeywordList
```
Comment (samuel27m) [inline docker-compose/init-compose.sh:22]: You don't need to add it here, other than that this looks good 👍

Comment (samuel27m) [inline docker-compose/init-compose.sh:22]: You don't need to add it here, other than that this looks good 👍

**Review by samuel27m — CHANGES_REQUESTED** [review]

File: docker-compose/init-compose.sh (hunk 1 above)
Comment (ktrops) [reply docker-compose/init-compose.sh:22]: init-compose.sh is used if we want to setup the docker containers, but run iwf-server locally to test local changes. If some one runs the server locally then runs java-sdk tests against it the tests will fail.

Comment (samuel27m) [reply docker-compose/init-compose.sh:22]: Ah I see, makes sense! Thanks for clarifying, appreciate it! 🙌

**Review by samuel27m — APPROVED** [review]

**Pull request merged**